LayoutKit.xcodeproj/project.pbxproj

```diff
@@ -1908,6 +1914,7 @@
 				SKIP_INSTALL = YES;
 				SWIFT_ACTIVE_COMPILATION_CONDITIONS = DEBUG;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
+				SWIFT_VERSION = 5.0;
```

Comment:
if swift version is set at the project/workspace level, it should be inherited into all of these configurations - these additions can probably be deleted.

Reply:
Yes, you are right. Removed these unnecessary changes.